massdatasets.py

```diff
@@ -113,3 +143,75 @@ def __init__(self,
                  base_path='/vol/vssp/datasets/audio/MSD100',
                  xlsx_name='msd100.xlsx'):
         super(MSD100, self).__init__(base_path, xlsx_name)
+
+
+class MUS2016(Dataset):
+    '''
+    The naming conventions are strange I know.
+    Basically MUS2016 is correct, but the data files we have received and are
+    available for download use 2017.
+    '''
+
+    def __init__(self,
+                 base_path='/vol/vssp/maruss/data2/MUS2017',
+                 results_filename='sisec_mus_2017_full.csv'):
+        super(MUS2016, self).__init__(base_path)
```

Comment:
In [python 3](https://docs.python.org/3.0/library/functions.html#super) you can now also write:
```python
super().__init__(base_path)
```
which I find easier to read.

Reply:
Yeah but what about the old guys? Though I haven't even tested with Python 2.x yet...

Reply:
OK, I abandoned Python 2 ;)